src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/JudicialUsersFunctionalTest.java

```diff
@@ -21,70 +25,73 @@
 
 import static java.lang.System.getenv;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.springframework.http.HttpStatus.FORBIDDEN;
 import static org.springframework.http.HttpStatus.NOT_FOUND;
 import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.hmcts.reform.judicialapi.util.JrdConstant.USER_DATA_NOT_FOUND;
+import static uk.gov.hmcts.reform.judicialapi.util.constants.SqlQueryConstant.DELETE_TEST_USERS_APPOINTMENT;
+import static uk.gov.hmcts.reform.judicialapi.util.constants.SqlQueryConstant.DELETE_TEST_USERS_AUTHORISATION;
+import static uk.gov.hmcts.reform.judicialapi.util.constants.SqlQueryConstant.DELETE_TEST_USERS_PROFILE;
+import static uk.gov.hmcts.reform.judicialapi.util.constants.SqlQueryConstant.INSERT_JUD_OFFICE_APPOINTMENT;
+import static uk.gov.hmcts.reform.judicialapi.util.constants.SqlQueryConstant.INSERT_JUD_OFFICE_AUTHORISATION;
+import static uk.gov.hmcts.reform.judicialapi.util.constants.SqlQueryConstant.INSERT_JUD_USER_PROFILES;
 
 @RunWith(CustomSerenityRunner.class)
 @WithTags({@WithTag("testType:Functional")})
 @Slf4j
 public class JudicialUsersFunctionalTest extends AuthorizationFunctionalTest {
 
-    private DataSource dataSource;
-    private JdbcTemplate template;
-
-    private static final String INSERT_MULTIPLE_USERS =
-              "INSERT INTO judicial_user_profile (per_id, personal_code, appointment, known_as, surname, full_name, "
-            + "ejudiciary_email, extracted_date, sidam_id) "
-            + "VALUES (12344, 'Emp1', 'Magistrate', 'Test', 'Test1','Test Test1', 'abc@gmail.com', current_timestamp, "
-            + "'44862987-4b00-e2e7-4ff8-281b87f16bf9') ON CONFLICT (per_id) DO NOTHING;";
-
-    private static final String DELETE_TEST_USERS = "DELETE FROM judicial_user_profile WHERE "
-            + "sidam_id IN ('44862987-4b00-e2e7-4ff8-281b87f16bf9');";
+    private static JdbcTemplate template;
 
     public static final String FETCH_USERS = "JrdUsersController.fetchUsers";
+    public static final String USERS_SEARCH = "JrdUsersController.searchUsers";
 
-    public void cleanUp() {
-        template.update(DELETE_TEST_USERS);
-        try {
-            template.getDataSource().getConnection().close();
-        } catch (Exception e) {
-            log.info("DB connection not found");
+    @BeforeClass
+    public static void setUp() {
```

Comment:
Can this functionality / test case which requires test data creation moved to integration test cases ? Reason being if something fails abruptly and some how @before method is not called the data will reside in AAT db.

Reply:
This is existing code which I have moved to @Before instead of adding it in all the test methods. Re work would need some effort to fix the existing tests as well

Reply:
ok wasn't knowing it is existing test cases but the approach needs to be corrected. Wouldnt will it be simple to remove all these functional test cases if integration test cases are already there ?

Reply:
We can remove it but there won't be any functional test case at all. Not sure if QA would be ok if there are no functional test reports

Reply:
you can create one test case with not data found, and rest all scenarios would be in integration tc

Reply:
this will be addressed in https://tools.hmcts.net/jira/browse/RDCC-3347